gma/src_java/com/google/firebase/gma/internal/cpp/BannerViewHelper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google LLC
+ * Copyright 2021 Google LLC
```

Comment:
Why update the copyright dates? Just curious.

Reply:
Figured that they were technically new files since old Admob is going back into the source base.